parkerbane/MailBux-FrontEnd
# #170107009 Fix PNG Issue On Safari

#### What does this PR do?
Fixes the bug that made PNGs appear black on Safari browser in MacOS.

#### How should this be manually tested?
Open with Safari on a Mac and confirm all images show correctly, especially the image on the signup page.

#### What are the relevant pivotal tracker stories?
[#170107009](https://www.pivotaltracker.com/story/show/170107009)
